requirements/deploy.txt

```diff
@@ -174,7 +174,7 @@ djangorestframework==3.14.0
     #   drf-extensions
 djangorestframework-jsonp==1.0.2
     # via -r requirements/pip.txt
-docker==6.0.1
+docker==6.1.0
```

Comment:
* https://github.com/docker/docker-py/releases/tag/6.1.0
* https://github.com/docker/docker-py/releases/tag/6.1.1

It seems fine.